src/yasa/evaluation.py

```diff
@@ -122,12 +137,23 @@ class EpochByEpochAgreement:
     .. plot::
 
         >>> import matplotlib.pyplot as plt
-        >>> fig, ax = plt.subplots(figsize=(6, 3), constrained_layout=True)
+        >>> fig, ax = plt.subplots(
+        ...     figsize=(
```

Comment:
Side effect of ruff, this is annoying

Reply:
Try adjusting the ruff numpy docstring length setting in toml? Maybe you can set it long or disable entirely.